galaxy/ftp.md

```diff
@@ -4,9 +4,21 @@ title: FTP service instructions
 permalink: /ftp/
 ---
 
-# UseGalaxy.eu FTP Server
+-------
+
+## We need your help and feedback ✨ 
+
+We integrated a new way to upload data into Galaxy based on the [tus.io](https://tus.io/). This will enable our users to upload
+large amounts of data via the web-interface in a more reliable way and with resumable file upload support. This also means that the 1GB limit
+for uploads via your browser does __not__ exist anmore ✨ . If this scales well over the next month and you are happy, we would like to retire
+the FTP server and encourage all our users to use the normal upload tool in Galaxy.
+
+Try our the new upload and [let us know how it goes]](mailto:galaxy@informatik.uni-freiburg.de?subject=Tus.io+feedback).
```

Comment:
contact@usegalaxy.eu?

Reply:
Mh yes, took it from the page further below. Need to change both I think.